### ISSUE TITLE
Everviz failing on RHEL6

### ISSUE THREAD

**Issue by oyvindeide** [equinor/everviz#31]
```
Traceback (most recent call last):
  File "/private/oyveid/project/everviz/everviz/everest_hooks.py", line 27, in visualize_data
    config = webviz_config(api)
  File "/private/oyveid/project/everviz/everviz/config.py", line 17, in webviz_config
    summary_values.page_layout(api),
  File "/private/oyveid/project/everviz/everviz/pages/summary_values.py", line 73, in page_layout
    sources = _set_up_data_sources(api)
  File "/private/oyveid/project/everviz/everviz/pages/summary_values.py", line 65, in _set_up_data_sources
    data = _summary_values(summary_values)
  File "/private/oyveid/project/everviz/everviz/pages/summary_values.py", line 50, in _summary_values
    summary_values.columns = ["Mean", "P10", "P90"]
  File "/shared/python/lib/python3.6/site-packages/pandas/core/generic.py", line 5080, in __setattr__
    return object.__setattr__(self, name, value)
  File "pandas/_libs/properties.pyx", line 69, in pandas._libs.properties.AxisProperty.__set__
  File "/shared/python/lib/python3.6/site-packages/pandas/core/generic.py", line 638, in _set_axis
    self._data.set_axis(axis, labels)
  File "/shared/python/lib/python3.6/site-packages/pandas/core/internals/managers.py", line 155, in set_axis
    'values have {new} elements'.format(old=old_len, new=new_len))
ValueError: Length mismatch: Expected axis has 0 elements, new values have 3 elements
```
This is probably caused by using an older version of `pandas`, as more recent versions are incompatible with RHEL6.

**Comment by DanSava** [comment]
Not really sure it is related to `pandas` version

I also bumped into the same error message when the optimization case does not have any summary data.

**Comment by oyvindeide** [comment]
Yes, figured that was the problem, will add some checks

**Comment by oyvindeide** [comment]
Solved by: #32
